packages/gluestick/src/commands/destroy.js

```diff
@@ -95,15 +95,23 @@ module.exports = async (context: Context, command: Command, name: string, option
   // If we destroyed a reducer, remove it from the reducers index
   if (command === 'reducer') {
     const reducerIndexPath = path.join(CWD, 'src', entryPoint, 'reducers', 'index.js');
+    // console.log(dirname, basename);
     try {
       const indexLines = fs.readFileSync(reducerIndexPath, { encoding: 'utf8' }).split('\n');
-      const reducerLine = `export { default as ${generatedFileName} } from "./${generatedFileName}"`;
-      const newIndexLines = indexLines.filter(indexLine =>
-        // Only return lines from the reducer index that do not return the reducer
-        // we just destroyed. Check for semi colon and non-semi colon lines
-         indexLine !== reducerLine && indexLine !== `${reducerLine};`);
+      const reducerName: string = dirname === '.' ? basename : `${
+        convertToCamelCase(dirname.replace('/', ''))
+      }${
+        basename[0].toUpperCase()}${basename.slice(1)
```

Comment:
Maybe can we move this functionality to helper? Its 3rd time when i see the same code :)

What you think?